src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -295,6 +305,18 @@ public function configureOptions(OptionsResolver $resolver)
             return $options['expanded'];
         };
 
+        $choiceTranslationDomain = function (Options $options) {
+            return $options['translation_domain'];
+        };
+
+        $choiceTranslationDomainNormalizer = function (Options $options, $choiceTranslationDomain) {
+            if (true === $choiceTranslationDomain) {
+                return;
```

Comment:
so just change the normalizer to return `$options['translation_domain']` when true.
and then change the default value for `choice`_translation_domain` to true instead of the lazy option. change buildview to handle true instead of null. change the upgrade to use `true`.